src/demo/index.html

```diff
@@ -628,6 +638,29 @@ <h1>Symphony Electron API Demo</h1>
             activityTimer = startActivityInterval();
         };
 
+        /**
+         * Get CPU usage
+         */
+        const cpuMessurement = document.getElementById('button-get-cpu-usage');
```

Comment:
typo: please refactor the variable to `cpuMeasurement`

Reply:
Fixed